force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -260,10 +354,16 @@ public with sharing class VariablesDataTypesOperators {
     public static Integer performDivisionAndCast(Double a, Double b) {
         // Perform the division and cast (round down) off the result.
         Double divisionResult = null;
+        if (a==null|| b==null) {
+            return null;
+            
+        } else {
+            divisionResult = a / b;
+        }
 
         // Write the code for type casting the divisionResult to an Integer
-        Integer roundedResult = null;
-        return null; // Replace null with the variable you used to store the result
+        Integer roundedResult = divisionResult.intValue();   
```

Comment:
You can also cast like this:
`Integer roundedResult = (Integer) divisionResult;`